### ISSUE TITLE
Please add OpenGL ES 3.0 And ARMv7l Support!

### ISSUE THREAD

**Issue by SiegemusDaBoss** [SachinVin/citra_android#221]
I have a Samsung Galaxy Tab A 8.0 2017 16GB tablet, and I would like to know why as soon as I open a ROM added to Citra, it crashes. My tablet has OpenGL ES 3.0 support, and an Adreno 308 GPU, with ARMv7l support. According to my research, the Nintendo 3DS uses the PICA200 GPU, which uses OpenGL ES 2.0. I am a COMPLETE NOOB when it comes to software development, so I would appreciate it if all explanations/responses to this issue would be in layman's terms. My Samsung Galaxy Tab A 8.0 2017 16GB is running the updated Android 8.1.0 OS. Please, if it's possible, SachinVin, add support for ARMv7l & 32-bit processors and GPU's. Also, if possible, please make Citra for Android run on devices that support OpenGL ES 3.0 and lower, since OpenGL ES 3.0 is engineered to have backward compatibility with OpenGL ES 2.0, which, as I've mentioned, is what the Nintendo 3DS's PICA200 processor/and or GPU runs on. Also, I am not in a position where I can just go out and buy another phone that probably costs over US $800. Thank you in advance, and if you can't add OpenGL ES 3.0 and ARMv7l support as I requested in this comment, I 100% understand. Have a great day!

**Comment by CyberRedX** [comment]
Hi @SiegemusDaBoss @SachinVin I am also facing the same issue-ish, I am running citra on Android lollipop but whenever I open a rom the app crashes. Please add support for ARMv7 and opengl es 2.0 please, Thank u
